From 0b55661d5854e4677f5257ef859cdb6f420a93f2 Mon Sep 17 00:00:00 2001
From: JinSen Chen <kevin.chen@rock-chips.com>
Date: Tue, 27 Mar 2018 19:28:42 +0800
Subject: [PATCH] ARM: dts: rk3288-evb-android-rk808-edp: update regulators
 node

set the regulator named "vcc_wl" stay on in suspend.

Change-Id: I6d21b8904a6367938a386df8e905ede62f4069a8
Signed-off-by: JinSen Chen <kevin.chen@rock-chips.com>
---
 arch/arm/boot/dts/rk3288-evb-android-rk808-edp.dts | 4 ++--
 arch/arm/boot/dts/rk3288-evb.dtsi                  | 4 +---
 2 files changed, 3 insertions(+), 5 deletions(-)

diff --git a/arch/arm/boot/dts/rk3288-evb-android-rk808-edp.dts b/arch/arm/boot/dts/rk3288-evb-android-rk808-edp.dts
index ad913531c56b..32ba05b64494 100644
--- a/arch/arm/boot/dts/rk3288-evb-android-rk808-edp.dts
+++ b/arch/arm/boot/dts/rk3288-evb-android-rk808-edp.dts
@@ -260,10 +260,10 @@
 				regulator-always-on;
 				regulator-boot-on;
 				regulator-min-microvolt = <1800000>;
-				regulator-max-microvolt = <3300000>;
+				regulator-max-microvolt = <1800000>;
 				regulator-name = "vcc_wl";
 				regulator-state-mem {
-					regulator-off-in-suspend;
+					regulator-on-in-suspend;
 				};
 			};
 
diff --git a/arch/arm/boot/dts/rk3288-evb.dtsi b/arch/arm/boot/dts/rk3288-evb.dtsi
index 1101fe5c64d4..43ed2fed00f9 100644
--- a/arch/arm/boot/dts/rk3288-evb.dtsi
+++ b/arch/arm/boot/dts/rk3288-evb.dtsi
@@ -363,9 +363,7 @@
 
 &sdio0 {
 	status = "okay";
-
-	clock-frequency = <50000000>;
-	clock-freq-min-max = <200000 50000000>;
+	max-frequency = <150000000>;
 
 	bus-width = <4>;
 	cap-sd-highspeed;
-- 
2.35.3

